Example/ExampleTests/KZPropertyMapperTests.m

```diff
@@ -93,6 +93,20 @@
           [KZPropertyMapper mapValuesFrom:sourceDictionary toInstance:testObject usingMapping:mapping];
           [[testObject.contentURL should] equal:[NSURL URLWithString:@"http://test.com/video.mp4"]];
         });
+
+        it(@"should handle nil from any boxing", ^{
+          sourceDictionary = @{@"videoURL" : [NSNull null]};
```

Comment:
this isn't really using boxing though?

Reply:
The `videoURL`  property on the `TestObject` uses the buildin NSURL-boxing

Reply:
ah right